lib/connectors_sdk/share_point/http_call_wrapper.rb

```diff
@@ -23,10 +23,10 @@ def extractor(params)
         # XXX can we cache that class across calls?
         ConnectorsSdk::SharePoint::Extractor.new(
           content_source_id: BSON::ObjectId.new,
-          service_type: 'sharepoint_online',
+          service_type: SERVICE_TYPE,
```

Comment:
`SERVICE_TYPE` is `sharepoint`, I don't think it make any more sense to use `sharepoint_online` here.